README.rst

```diff
@@ -34,6 +34,14 @@ In the Kinto project settings
     kinto.attachment.folder = {bucket_id}/{collection_id}
     kinto.attachment.keep_old_files = true
 
+
+If you want uploaded files to get gzipped when stored:
+
+.. code-block:: ini
+
+    kinto.attachment.gzipped = true
```

Comment:
Since the option can be controlled with the querystring, we could use a formulation in that section that says something like `If you want to **force** gzip to every collections`...